content/python/concepts/iterators/terms/list/list.md

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
While I understand the place of this entry file, I think we should move `list/list.md` from `iterator/terms/` to `built-in-functions/terms/`.